lang/po/Firmware_pl.po

```diff
@@ -2568,6 +2568,11 @@ msgstr "Załaduj nowy filament lub wyładuj poprzedni."
 msgid "Sensitivity"
 msgstr "Wrażliwość"
 
+#. MSG_MBL_FAILED_Z_CAL c=20 r=4
+#: ../../Firmware/Marlin_main.cpp:2976
+msgid "Mesh bed leveling failed. Please run Z calibration."
+msgstr "Poziomowanie stolu. Proszę uruchomić kalibrację Z."
```

Comment:
Is 63 characters ok and is "ł" available? If yes/yes then:

"Poziomowanie stołu nieudane. Proszę uruchomić kalibrację Z."

"Poziomowanie stołu" means "bed leveling". "nieudane" means "failed".

Reply:
@arekm Thanks for your comment. At this moment we replace all diacritics with aA-zZ chars but we want to keep the po files with these. So "ł" is not shown on LCD (at this moment see https://github.com/prusa3d/Prusa-Firmware/pull/3383) but appreciated to have these in the po files.
Auto correction in PO editors works way better this way.